### DISCUSSION TITLE
WebSocket Support for GraphQL Hive Gateway

### DISCUSSION THREAD

**Discussion by jamesmmchugh** [graphql-hive/console#5686] [Feature request]
Currently the the Gateway only supports SSE between the client and Gateway (though does support WS between the Gateway and the Subgraph).

Are there plans to add WebSocket support in the future to make this comparable with v0 of GraphQL Mesh?

**Comment by ardatan** [comment]
You can track the progress here;
https://github.com/ardatan/graphql-mesh/pull/7766
